contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -11,6 +11,13 @@ RETURNS ntext
 AS 'babelfishpg_tsql', 'tsql_query_to_xml_text'
 LANGUAGE C IMMUTABLE STRICT COST 100;
 
+-- Helper function to support the FOR JSON clause
+CREATE OR REPLACE FUNCTION sys.tsql_query_to_json_text(query text, mode int, include_null_value boolean,
+           without_array_wrappers boolean,root_name_present boolean, root_name text)
+RETURNS ntext
```

Comment:
Why do we use `ntext`?

Reply:
will change it to sys.NVARCHAR

Reply:
In that case, please also consider a proper typmod, e.g., sys.nvarchar(4000) or sys.nvarchar(max).